indexer/services/roundtable/src/tasks/subaccount-username-generator.ts

```diff
@@ -0,0 +1,34 @@
+import { logger } from '@dydxprotocol-indexer/base';
+import {
+  SubaccountUsernamesTable,
+  SubaccountsWithoutUsernamesResult,
+} from '@dydxprotocol-indexer/postgres';
+import { generateUsername } from 'unique-username-generator';
+
+const RANDOM_DIGITS = 3;
```

Comment:
nit: move this into config.ts, just in case we want to update it

SUBACCOUNT_USERNAME_NUM_RANDOM_DIGITS